.buildkite/scripts/dra-workflow.trigger.sh

```diff
@@ -2,10 +2,34 @@
 
 set -euo pipefail
 
-echo "steps:"
-
 source .buildkite/scripts/branches.sh
 
+# We use that filtering to keep different schedule for different branches
```

Comment:
I wonder if we need this. isn't that only used for triggering a DRA build from an intake? If we have something to commit to 7.17, we might want to keep building DRA artifacts immediately?

Reply:
scratch that.